scripts/release/debian/build.sh

```diff
@@ -21,17 +21,7 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # Update APT packages
 apt-get update
 # uuid-dev is used to build _uuid module: https://github.com/python/cpython/pull/3796
-apt-get install -y libssl-dev libffi-dev python3-dev zlib1g-dev uuid-dev wget
-
-# In Ubuntu 18.04, debhelper 11.1.6 has bug which makes it fail to dpkg-buildpackage. Use backport version instead.
-# https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=897569
-if cat /etc/lsb-release | grep 18.04
-then
-  apt-get install -y debhelper -t bionic-backports
-else
-  apt-get install -y debhelper
-fi
```

Comment:
This workaround was added in #24180. We also had a discussion at https://github.com/Azure/azure-cli/pull/24180#discussion_r1051834482. Glad we are finally getting rid of it.